### ISSUE TITLE
Firefox doesn't detect online/offline on Windows

### ISSUE THREAD

**Issue by WietseWind** [WietseWind/XRPL-MultiSignTool#6]
As mentioned in #5 (waiting for OS + version to try to reproduce this).

**Comment by popnop** [comment]
Windows 10


בתאריך יום ב׳, 24 ביוני 2019, 11:47, מאת Wietse Wind ‏<
notifications@github.com>:

**Comment by popnop** [comment]
do you need any more info ? 

i have one more question :) 

for security purpose if i have 3 accounts with fund  my set up is  
account 1 uses  2-3-4 ( 2 of 3) 
account 2 uses  1-3-4 ( 2 of 3) 
account 3 uses  1-2-4 ( 2 of 3) 
 and i  **disable account 1-2-3  master keys**  
**i can still use the disabled master keys  for multi signing** 
this is correct?

**Comment by WietseWind** [comment]
@popnop Yes, got it. Will try to reproduce. You can try it with TestNet :)
